lms/static/scripts/frontend_apps/components/dashboard/StudentsActivity.tsx

```diff
@@ -31,34 +28,33 @@ export default function StudentsActivity() {
   );
 
   const title = `Assignment: ${assignment.data?.title}`;
-  const [order, setOrder] = useState<MandatoryOrder<StudentStats>>({
-    field: 'display_name',
-    direction: 'ascending',
-  });
-  const orderedStudents = useOrderedRows(students.data ?? [], order);
 
   return (
     <Card>
+      <CardHeader fullWidth>
+        <CardTitle tagName="h2">
+          <span data-testid="title">
```

Comment:
Can the `data-testid` go on the parent `CardTitle` component, avoiding the extra `<span>`?

Reply:
Yep, it can be done.

This went through a couple different stages, and in one of them this was a prop for some other component. That's why I added an explicit wrapper.

But with current approach, it is definitely not needed.